code/sensors/bme_imu.py

```diff
@@ -0,0 +1,24 @@
+import smbus2
+import bme280
+from time import sleep
+
+address = 0x77
+i2c = smbus2.SMBus(1)
+log_path_temp = "/home/pi/sensor_logs/bme_imu_temp.txt"
```

Comment:
these paths should be configurable from script arguments, please refer to [argparse](https://docs.python.org/3/library/argparse.html)